Include the sceptre-resolver-cmd

zxiiro opened the pull request:
This includes the sceptre-resolver-cmd
(https://github.com/Sceptre/sceptre-resolver-cmd) into Sceptre as
a core resolver.

Signed-off-by: Thanh Ha <zxiiro@gmail.com>

depends on https://github.com/Sceptre/sceptre-resolver-cmd/pull/5

zaro0508 commented on setup.py:
```diff
@@ -28,6 +28,7 @@ def get_version(rel_path):
     "Jinja2>=2.8,<3",
     "colorama>=0.3.9",
     "packaging>=16.8,<17.0",
+    "sceptre-file-resolver>=1.1.2,<2",
```
this is not the cmd resolver

zxiiro replied:
Woops. I updated the version but forgot the name. Fixed.

zaro0508 commented:
there is a [release for sceptre-resolver-cmd](https://pypi.org/project/sceptre-cmd-resolver/) now @zxiiro, would you like to finish this up?

zxiiro commented:
> there is a [release for sceptre-cmd-resolver](https://pypi.org/project/sceptre-cmd-resolver/) now @zxiiro, would you like to finish this up?

Rebased and updated to 1.1.3.
